drivers/tty/serdev/pibridge.c

```diff
@@ -175,12 +176,14 @@ int pibridge_recv_timeout(u8 *buf, u16 len, u16 timeout)
 				len, ret, jiffies, kfifo_len(&pi->read_fifo));
 	return ret;
 }
+EXPORT_SYMBOL(pibridge_recv_timeout);
 
 int pibridge_recv(u8 *buf, u16 len)
 {
 	/* using default timeout REV_PI_IO_TIMEOUT */
 	return pibridge_recv_timeout(buf, len, REV_PI_IO_TIMEOUT);
 }
+EXPORT_SYMBOL(pibridge_recv);
```

Comment:
it maybe not needed for pibridge_recv to export, as pibridge_recv_timeout should be enough

Reply:
For now I only made use of the low-level functions `pibridge_send()` and `pibridge_recv_timeout()` (the only exception is `pibridge_req_gate_tmt()` since I found it easier to replace the current code in this case). However this was only to have as minimal impact (i.e. changes) to the current code as possible. I do not think that the high-level functions are worthless though, maybe we can use it later when we implement the new piControl design.